src/global-styles.scss

```diff
@@ -1,14 +1,71 @@
-$example-color: cornflowerblue;
+// gray-700
+// $bg-color: #374151;
 
-.example-class {
-    color: $example-color;
+// zinc-900
+$bg-color: #18181b;
+$text-color: #f8fafc;
+
+// gray-800
+// $card-lv-1-color: #1f2937;
+
+// zinc-700
+$card-lv-1-color: #3f3f46;
+
+
+
+html {
+    background-color: $bg-color;
+    color: $text-color;
+    height: 100%;
+}
+
+body {
+    height: 100%;
+}
+
+.sveltekit-container {
+    height: 100%;
 }
 
 .button {
+    all: unset;
     padding: 8px;
     border-radius: 4px;
     border: 2px solid #999;
     text-decoration: none;
     cursor: pointer;
+    user-select: none;
     display: inline-block;
-}
\ No newline at end of file
+}
+
+a, a:visited, a:hover, a:active {
+    color: inherit;
+}
+
+// temporary class, pending addition of full size styling.
+// When you're on desktop viewport size, it will confine all UI to a mobile-sized area
+.mobile-sized-container {
```

Comment:
I don't think we'll need this.  Look into the element size units vh and vw.  Combined with flex boxes, you get all the dynamic sizing and restructuring you could want.  I usually only use max-width and max-height + pixel sixes for things like text (width over 600px becomes hard to read and is an accessibility thing) or when working with pixel images that I want to keep true to original size.

Reply:
You're saying we should just style things responsively, to both desktop and mobile, from the get go?

I might resist.  We may choose to do big restructures early on.  

Confining it to a mobile size makes it easy on the eyes during development, and I only have to think about one type of screen orientation.  This feels nice and efficient when we might have to slice and dice the UI a few times in the early iteration(s).

Reply:
Another idea would be to start by focusing only on the desktop view, to the exclusion of mobile view.  I only went mobile-first because I was at a recent meetup where a guy recommended designing mobile-first.

My high level goal is for CSS to be out of my way as much as possible during the early phases of development.